doc/source/multivariate.rst

```diff
@@ -144,12 +144,12 @@ We realize that the mean vector and the covariance often have special forms in p
 
 .. code-block:: julia
 
-    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector{Float64},ZeroVector{Float64}}} <: AbstractMvNormal
+    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector,ZeroVector}} <: AbstractMvNormal
```

Comment:
That's a good example. However, it would complicate the type significantly since it would require two element type parameters for full dispatch generality so, at least initially until a demand appears, I think it would be better to restrict the parameters.